app/src/main/java/org/pesmypetcare/mypetcare/utilities/ImageManager.java

```diff
@@ -85,7 +87,13 @@ public static void writeImage(String relativePath, String fileName, byte[] bytes
         }
 
         byte[] bytes = convertToByteArray(bytesList);
-        input.close();
+        input.close();*/
+
+        String imagePath = PATH + File.separator + relativePath + File.separator + imageName + EXTENSION;
+        FileChannel channel = new FileInputStream(imagePath).getChannel();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Ensure that resources like this FileChannel object are closed after use](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5310682)